x/supplier/client/config/supplier_configs_reader_test.go

```diff
@@ -0,0 +1,308 @@
+package config_test
```

Comment:
I only skimmed through the tests (didn't review line by line) bu going to assume we have good coverage, ty!